corehq/apps/cloudcare/static/cloudcare/js/form_entry/const.js

```diff
@@ -1,7 +1,6 @@
 'use strict';
 hqDefine("cloudcare/js/form_entry/const", [], function () {
     return {
-        ADD_GROUP_TYPE: 'add-group',
```

Comment:
Based on the commit comment, I imagine this whole commit was a beast to figure out.